extension/encoding/azureencodingextension/internal/unmarshaler/logs/config.go

```diff
@@ -4,11 +4,21 @@
 package logs // import "github.com/open-telemetry/opentelemetry-collector-contrib/extension/encoding/azureencodingextension/internal/unmarshaler/logs"
 
 type LogsConfig struct {
-	// TimeFormats is a list of time formats parsing layouts for Azure Traces Records
+	// TimeFormats is a list of time formats parsing layouts for Azure Resource Log Records
 	TimeFormats       []string `mapstructure:"time_formats"`
 	IncludeCategories []string `mapstructure:"include_categories"`
 	ExcludeCategories []string `mapstructure:"exclude_categories"`
 
 	// prevent unkeyed literal initialization
 	_ struct{}
 }
+
+// DefaultTimeFormats is a list of well-known time formats parsing layouts,
+// that was detected for supported Azure Resource Log Categories
+// It is used as a default timestamp parsing formats to simplify configuration
+// of logs parser for end-users
+var DefaultTimeFormats = []string{
```

Comment:
```suggestion
func DefaultTimeFormats() []string{
```

Nit: let's use a function to avoid global state.

Reply:
done